docs/xdr/features/collect/ingestion_methods/syslog/sekoiaio_forwarder.md

```diff
@@ -232,6 +232,98 @@ pull_policy: always
+
+Activating TLS requires setting up an encryption key, a certificate, and a CA certificate. These should be placed in the `./certs` directory within the forwarder's directory.
+Therefore it is necessary to mount this volume. Add the following line to the configuration:
+
+```yaml
+volumes:
+    - ./certs:/certs
+    [...]
+```
+
+### Creating the Key and Certificate
+
+We will now create the key and certificate that will be used for encryption. In this case, we will create a self-signed certificate, meaning that the server certificate and the CA (Certificate Authority) certificate will be the same. If you have expertise in managing certificates, you can create a certificate signed by a real CA.
+
+**Step one**: Create the directory and navigate to it:
+
+```
+mkdir certs && cd certs
+```
+
+**Step two**: Install OpenSSL
+
+=== "Debian, Ubuntu"
+
+    ```bash
+    sudo apt update
+    sudo apt install -y openssl
+    ```
+
+=== "Fedora, Red Hat, CentOS (yum)"
+
+    ```bash
+    sudo yum update
+    sudo yum install -y openssl
+    ```
+
+=== "Fedora, Red Hat, CentOS (dnf)"
+    
+    ```bash
+    sudo dnf update
+    sudo dnf install -y openssl
+    ```
+
+Step three: Create the key and certificate
+
+```
+openssl req -x509 \
+            -sha256 -days 1825 \
+            -nodes \
+            -newkey rsa:4096 \
+            -keyout server.key -out server.crt
+```
+
+- `openssl req`: Launches the process to create a Certificate Signing Request (CSR) or a self-signed certificate.
+- `-x509`: This option tells OpenSSL to generate a self-signed certificate rather than a CSR.
+- `-sha256`: Specifies that the SHA-256 hashing algorithm should be used for the certificate's signature.
+- `-days 1825`: Sets the certificate's validity period to 1825 days, which corresponds 5 years.
+- `-nodes`: This stands for "no DES" and indicates that the private key should not be encrypted, meaning no password will be required to use the private key.
+- `-newkey rsa:4096`: Generates a new RSA key pair of 4096 bits in length, and simultaneously creates a certificate request using this key pair.
+- `-keyout server.key`: Specifies the file where the generated private key should be saved. In this case, it will be saved to `server.key`.
+- `-out server.crt`: Specifies the file where the generated self-signed certificate should be saved. In this case, it will be saved to `server.crt`.
+
+**Step four**: Change permissions on the files
+
+```
+chmod 600 server.key server.crt
+```
+
+### Configuration of the intakes.yaml File
+
+To use TLS, you need to specify `tls` in the protocol definition:
+
+```
+protocol: tls
+```
+
+By default, as soon as you specify the tls protocol, the forwarder will attempt to read the private key and certificate at /certs/server.key and /certs/server.crt.
+If you wish to specify other filenames, you can do so in the intake configuration:
+
+```
+[...]
+protocol: tls
+tls_key_name: mykey.pem
+tls_cert_name: mycert.crt
+tls_ca_name: myca.crt
```

Comment:
I think that the name of the certificate and key must match the one in the generation steps. However, users won't be able to know which files to put here correctly.

Reply:
It was to tell that you can use custom names instead of the default ones, but I agree with you. I modify it